apps/src/lib/levelbuilder/CourseOfferingEditor.jsx

```diff
@@ -328,7 +344,22 @@ export default function CourseOfferingEditor(props) {
           </select>
         </label>
       ))}
-
+      <h3>Professional Learning</h3>
+      Professional Learning Program
+      <HelpTip>
+        <p>
+          Link to page where they can go to learn more about the professional
+          learning program
+        </p>
+      </HelpTip>
+      <input
+        type="text"
```

Comment:
From Dani's comment, could you make this a dropdown instead with the two choices pulled in from the backend?